### ISSUE TITLE
Migrating Modern Desktop application document is no longer correct

### ISSUE THREAD

**Issue by steveharter** [dotnet/docs#32169]
The [Migrating Modern Desktop applications](https://learn.microsoft.com/en-us/dotnet/architecture/modernize-desktop/migrate-modern-applications#migrating-configuration-files) has an example of `<configSections>` for "system.diagnostics" section that is no longer correct with v7. That section is now implicit and if specified will cause an exception.

The section starts with: "You may come across some issues when using configuration sections that were defined in the machine.config file. For example, consider the following configuration:"

See the breaking change at https://github.com/dotnet/docs/issues/32147

**Comment by erjain** [comment]
Hi @steveharter, thank you so much for sharing your findings. We are working on updating the docs with .NET 7 version. We will consider this issue while updating the document.

**Comment by erjain** [comment]
Hi @ardalis, can you take care of this?

**Comment by ardalis** [comment]
Yes, on it.

**Comment by erjain** [comment]
Hi @ardalis, are we good to close this?

**Comment by ardalis** [comment]
No, this was added later to the PO. I'll try to knock it out this week.
